lib/modules/manager/github-actions/community.ts

```diff
@@ -0,0 +1,118 @@
+import { z } from 'zod';
+
+import { escapeRegExp, regEx } from '../../../util/regex';
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+import { NpmDatasource } from '../../datasource/npm';
+import { PypiDatasource } from '../../datasource/pypi';
+import * as condaVersioning from '../../versioning/conda';
+import * as npmVersioning from '../../versioning/npm';
+import * as pep440versioning from '../../versioning/pep440';
+
+import type { PackageDependency } from '../types';
+
+function matchFullUrl(domain: string, action: string): z.ZodString {
+  return z
+    .string()
+    .regex(
+      regEx(
+        escapeRegExp('https://' + domain + '/') +
```

Comment:
use string templates

Reply:
do you mean a nexted string template ``  `${escapeRegExp(`https://${domain}/`)}${escapeRegExp(action)}(@.+)?$` ``?